### ISSUE TITLE
Messages not Loading automatically

### ISSUE THREAD

**Issue by liberaltech** [AravindVasudev/Babble#1]
I managed to get this working on Heroku but the page has to be reloaded before I can see any new messages. Is this the way the app was intended to work?

**Comment by AravindVasudev** [comment]
No, it uses socket.io so you should be receiving it instantaneously.  You can check its working here: [babble.gq](http://babble.gq). You can post your heroku link here if you don't mind and I'll check it out.

**Comment by liberaltech** [comment]
here you go [iBabble](https://ibabble.herokuapp.com)

**Comment by liberaltech** [comment]
I sent the last two messages on the babble.gq site you have but I had to reload the page to see them.

**Comment by liberaltech** [comment]
update: 
This also doesn't seem to be working on iOS at all. This could possibly be on my end only.

**Comment by liberaltech** [comment]
Tested it on Firefox and the problem doesn't seem to happen.
I think this is a safari issue, probably SSL to be more specific.

**Comment by AravindVasudev** [comment]
Just tested in my iPad again and found out I have the same issue too :sweat_smile: .
